optaplanner-quickstart/pom.xml

```diff
@@ -155,11 +169,9 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
-                  <systemPropertyVariables>
+                  <systemProperties>
```

Comment:
Why did you change that? IIRC `systemProperties` is deprecated.

Reply:
Fixing :)